installing/installing_azure_stack_hub/installing-azure-stack-hub-network-customizations.adoc

```diff
@@ -0,0 +1,85 @@
+[id="installing-azure-stack-hub-network-customizations"]
+= Installing a cluster on Azure Stack Hub with network customizations
+include::modules/common-attributes.adoc[]
```

Comment:
So I think this needs to be `_attributes/common-attributes.adoc` to work now.

Reply:
Update complete.